content/news/052/index.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "This Month in Rust GameDev #52 - April 2024"
+transparent = true
+date = 05.2024
```

Comment:
```suggestion
date = 03.05.2024
```

Reply:
I think the date needs to be in YYYY-MM-DD format? That's how it is for the existing files (and what the [Zola docs](https://www.getzola.org/documentation/content/page/) say).

Reply:
Addressed in https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1472